docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -36,6 +36,53 @@ Additional information, including the specific error and a stack trace, is provi
 [role="screenshot"]
 image::images/synthetic-app-error.png[Synthetics app error]
 
+Select a step name to view the waterfall chart for the synthetic monitor.
+
+[role="screenshot"]
+image::images/synthetics-waterfall.png[Synthetics waterfall chart]
+
+The waterfall chart provides a visualization of every request the page made when
+a user executed it. Each line in the chart represents an HTTP network request and
+helps you quickly identify what resources are taking the longest to load and in what
+order they are loading.
+
+The colored bars within each line indicate the time spent per resource. Each color
+represents a different part of that resource's loading process and
+includes the time spent downloading content for specific Multipurpose Internet Mail
+Extensions (MIME) types: HTML, JS, CSS, Media, Font, and Other.
+
+It's important to understand each phase of a request so you can improve your site's
+speed by reducing the time spent in each one.
+
+// This is collapsed by default
+[%collapsible]
+.HTTP request phases
+====
+Queued/Blocked::
+The request was initiated but is blocked or queued.
+
+DNS::
+The DNS lookup to convert the hostname to an IP Address.
+
+Connecting::
+The time it took the request to connect to the server, which could indicate network
```

Comment:
Connects are normal, however maybe point out that a slow connect could be for the reasons stated, for example:
`The time it took the request to connect to the server. Lengthy connections could indicate network
issues, connection errors, or an overloaded server.`

Reply:
Thanks.